core/iwasm/aot/aot_loader.c

```diff
@@ -1214,10 +1224,11 @@ load_types(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
 fail:
     return false;
 }
+#endif
```

Comment:
Had better add a comment, like `/* end of WASM_ENABLE_GC != 0 */`

Reply:
OK